Sources/Tuvi.Proton.Impl/ProtonClient.cs

```diff
@@ -164,7 +165,7 @@ internal class MessageMetadata
         public IReadOnlyList<EmailAddress> CCList { get; set; }
         public IReadOnlyList<EmailAddress> BCCList { get; set; }
         public IReadOnlyList<EmailAddress> ReplyTos { get; set; }
-        public MessageFlag Flags { get; set; }
+        //public MessageFlag Flags { get; set; }// TODO: failed to deserialize, flag values do not coincide with Proton's flag values
```

Comment:
What JSON do we receive?
look at: https://github.com/ProtonMail/go-proton-api/blob/master/message_types.go#L27